RankensteinApp/App.js

```diff
@@ -0,0 +1,183 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, TextInput, View, TouchableOpacity, Platform,} from 'react-native';
+import {createStackNavigator,} from 'react-navigation';
+
+
+
+class LoginScreen extends React.Component{
```

Comment:
Would it be possible to move this class out to a different file like with the CreateAccountScreen?